src/style.css

```diff
@@ -1 +1,137 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-family: Avenir, sans-serif;
+	
+	--text-color:  #3d4e61;
+	--border-color: #e1e7ed;
+}
+
+.big-form {
+  display: flex;
+	margin-top: 20px;
+	width: 100%;
+  height: 70px;
+}
+
+.big-form__label {
+	display: block;
+  position: relative;
+  width: 100%;
+}
+
+.big-form__label::after {
+  display: block;
+  position: absolute;
+  content: "";
+	background-image: url("./images/Search.svg");
+	background-repeat: no-repeat;
+	background-size: contain;
+	width: 19px;
+	height: 19px;
+	top: 25px;
+	left: 26px;
+}
+
+.big-form__input {
+	box-sizing: border-box;
+  font-weight: 300;
+	font-size: 16px;
+	line-height: 22px;
+  width: 100%;
+  height: 100%;
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  color: var(--text-color);
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+	padding-left: 62px;
+}
+
+.big-form__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.big-form__input::placeholder {
+	color:  var(--text-color);
+  font-weight: 300;
+	font-size: 16px;
+	line-height: 22px;
+}
+
+.big-form__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 600;
+	outline: none;
+}
+
+.small-form {
+  display: flex;
+	margin-top: 20px;
+	width: 100%;
+  height: 42px;
+}
+
+.small-form__label {
+	display: block;
+  position: relative;
+  width: 100%;
+}
+
+.small-form__label::after {
+  display: block;
+  position: absolute;
+  content: "";
+	background-image: url("./images/Search.svg");
+	background-repeat: no-repeat;
+	background-size: contain;
+	width: 11px;
+	height: 11px;
+	top: 15px;
+	left: 13px;
+}
+
+.small-form__input {
+	box-sizing: border-box;
+  font-weight: 300;
```

Comment:
You are repeating a lot of styles for small and big forms.

The way to do it correctly, w/o repeating yourself:
- Create a common class `form__input` and write there all the styles that are needed (for big input for example).
- Create `form__input--small` class and **overwrite** only those properties that are different in small input.
- For small input in HTML use both classes: common `form__input` , and modificator `form__input--small`

That's related to all elements